docs/.project/PATCH_RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,25 @@
+# Patch Release Checklist
+
+A list of things to perform with Camunda Modeler patch release.
+
+__Hint:__ Create a release issue and copy the template below into it.
+
+
+```markdown
+* [ ] make sure dependencies are released (`rm -rf node_modules && npm i && npm run all` works)
+    * `bpmn-js`, `dmn-js`, `cmmn-js`, `*-properties-panel`, `*-moddle`, ...
+* [ ] close all issues which are solved by dependency updates
+* [ ] smoke test to verify all diagrams can be created
+* [ ] update CHANGELOG
+* [ ] semantic release (`npm run release`), cf. [release schema](https://github.com/bpmn-io/internal-docs/tree/master/release-schema) 
+* [ ] wait for [Travis](https://travis-ci.org/camunda/camunda-modeler) to build the release
+* [ ] execute [integration test](https://github.com/camunda/camunda-modeler/blob/master/docs/.project/INTEGRATION_TEST.md) on [released artifacts](https://github.com/camunda/camunda-modeler/releases)
```

Comment:
Don't think we have to have integration testing and the qa team involved in every patch release, but let's keep this in if we want to have it in some cases.